Semaforos/Semaforo.cpp

```diff
@@ -46,8 +44,8 @@ int Semaforo :: p () const {
 
 	struct sembuf operacion;
 
-	operacion.sem_num = 0;	// numero de semaforo
-	operacion.sem_op  = -1;	// restar 1 al semaforo
+	operacion.sem_num = 0;	// numero de semaforoConCocinero
```

Comment:
jaja metí un search and replace de semaforo por semaforoConCocinero porque no se entendía el nombre de las variables en la clase Mozo o Cocinero y me lo cambió acá el forro